apps/web/src/css/global.css

```
@@ -1,6 +1,42 @@
+}
+
+@font-face {
+  font-family: 'AirbnbCerealBold';
+  src: url('/fonts/AirbnbCerealBold.woff') format('woff');
+  font-weight: 800;
+  font-style: normal;
+  font-display: swap;
+}
+
+.font-light {
+  font-family: 'AirbnbCerealLight', ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "AirbnbCerealLight" (font-family-name-quotes)

(font-family-name-quotes)